server/asset/bch/bch.go

```diff
@@ -107,8 +107,8 @@ func (bch *BCHBackend) Contract(coinID []byte, redeemScript []byte) (*asset.Cont
 
 // estimateFee estimates the network transaction fee rate using the estimatefee
 // RPC.
-func estimateFee(node btc.BTCNode) (uint64, error) {
-	resp, err := node.RawRequest("estimatefee", nil)
+func estimateFee(node btc.RPCClient) (uint64, error) {
+	resp, err := node.Requester.RawRequest(nil, "estimatefee", nil)
```

Comment:
I'm not %100 sure, but it looks like you could just change estimateFee to take the Requester and you would then not have to export btc.RPCClient as well.